e2e/scenario/scenario_mariner-gpu-azurecni.go

```diff
@@ -0,0 +1,36 @@
+package scenario
+
+import (
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice"
+)
+
+func marinergpu_azurecni() *Scenario {
```

Comment:
can you specify that we're using marinerv2 in both the file name and function name here?

checkout examples in other mariner scenario definitiosn